Dockerfile

```diff
@@ -1,8 +1,4 @@
 FROM gliderlabs/alpine:3.2
 RUN apk-install ca-certificates
 ENTRYPOINT ["/bin/marathon-stats"]
-
-WORKDIR /
```

Comment:
`WORKDIR` was originally added so `/bin/marathon-stats` could reliably find `aws_prices.csv` using a relative path.  Now that `aws_prices.csv` is gone, we no longer need this command.